lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -418,6 +418,7 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
 
   // Pass custom share links to the front end. Unique by UID.
   $base_url = url(base_path(), array('absolute'=> TRUE, 'language' => 'en-global'));
+  $campaign_path = url(current_path(), array('absolute' => TRUE));
   $country_prefix_for_share = (dosomething_global_get_current_prefix()) ? strtoupper(dosomething_global_get_current_prefix()) : 'global';
   $share_image = $base_url . 'sites/default/files/images/problem-' . $campaign->nid . '-' . $country_prefix_for_share . '.png';
```

Comment:
The share image is the image that comes up on facebook or tumblr, so currently for the custom social share buttons that's just the same image as if you used the problem share buttons at the top of the page. Do we want to put something else in there for now?